frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -21,7 +21,7 @@ const HealthItem: React.FC<HealthItemProps> = React.memo(
     return (
       <div
         className={classNames('co-status-card__health-item', className)}
-        data-test={`${title}-health-item`}
+        data-item-id={`${title.toLowerCase().replace(/\s/g, '-')}-health-item`}
```

Comment:
just small doubt, why we are using dataStatusID and data-item-id instead of data-test?

Reply:
This was discussed in a sync-up with @bipuladh a few days back.

Reply:
```suggestion
        data-item-id={`${title}-health-item`}
```